src/features/common/TreeCounter/TreeCounter.tsx

```diff
@@ -64,81 +88,63 @@ export default function TpoProfile(props: any) {
-              count: Number(props.planted),
-              treeCount: getFormattedNumber(
-                i18n.language,
-                Number(props.planted)
-              ),
-            })}
-          </h2>
-        </div>
+      {isHomeTreeCounter ? (
+        <ProfileCircularProgress value={progress} />
+      ) : (
+        <HomeCircularProgress value={progress} />
+      )}
 
-        {props.target ? (
-          <div className={treeCounterStyles.treeCounterDataField}>
-            <h1>
-              {localizedAbbreviatedNumber(
-                i18n.language,
-                Number(props.target),
-                1
-              )}
-            </h1>
-            <div className={treeCounterStyles.target}>
-              <h2
-                className={
-                  props.authenticatedType === 'public'
-                    ? treeCounterStyles.targetText
-                    : treeCounterStyles.targetTextForPrivate
-                }
-              >
-                {t('me:target')}
-              </h2>
-              {props.authenticatedType === 'private' && (
-                <button
-                  id={'treeCounterEdit'}
-                  className={treeCounterStyles.editTragetContainer}
-                  onClick={() => props.handleAddTargetModalOpen()}
-                >
-                  <EditIcon color="white"></EditIcon>
-                </button>
-              )}
-            </div>
-          </div>
-        ) : null}
+      <div
+        className={
+          isHomeTreeCounter
+            ? treeCounterStyles.backgroundCircle
+            : treeCounterStyles.backgroundCircleForTenant
+        }
+      />
 
-        {props.authenticatedType === 'private' && props.target === 0 && (
-          <button
-            id={'addTarget'}
-            onClick={() => props.handleAddTargetModalOpen()}
-            className={treeCounterStyles.addTargetButton}
-          >
-            <p
-              style={{
-                color:
-                  theme === 'theme-light'
-                    ? themeProperties.light.light
-                    : themeProperties.dark.dark,
-              }}
-            >
-              {t('me:addTarget')}{' '}
-            </p>
-          </button>
-        )}
-      </div>
+      {isHomeTreeCounter ? (
+        <HomeTreeCounter planted={props?.planted} target={props.target} />
+      ) : (
+        <div className={treeCounterStyles.treeCounterData}>
+          <div>
+            <PlantedTressBlackSvg color={'#4F4F4F'} />
+          </div>
+          <div className={treeCounterStyles.dataContainer}>
+            {props?.planted && (
+              <div>
+                {localizedAbbreviatedNumber(
+                  i18n.language,
+                  Number(props.planted - props.restoredAreaUnit),
+                  1
+                )}
+              </div>
+            )}
+            {props.target !== undefined && props.target !== 0 && (
+              <div>{'of'}</div>
```

Comment:
Regarding translation this is the worst way to develop some UI. Please change this towards something using a resource like `"treesOfTrees": "{{count1}} of {{count2}}"` if both props exists and have valid values. Otherwise you can still just show one of the values not using any translation resource.

Reply:
I fixed this myself while adding some more German translations for better testing.